File: 75. Sort Colors.cpp
class Solution {
public:
    void sortColors(vector<int>& arr) {
        int n=arr.size();
        int low=0;
        int mid=0;
        int high=n-1;
        while(mid<=high){
            if(arr[mid]==0){
               swap(arr[mid],arr[low]);
               arr[low]=0;
               mid++;
               low++;
            }
            else if(arr[mid]==1){
                mid++;
            }
            else{
                swap(arr[mid],arr[high]);
                arr[high]=2;
                high--;
            }
        }
        return;
    }
};

File: 75. Sort Colors.js
var sortColors = function(arr) {
        let n=arr.length;
        let low=0;
        let mid=0;
        let high=n-1;
        while(mid<=high){
            if(arr[mid]==0){
               arr[mid]=arr[low];
               arr[low]=0;
               mid++;
               low++;
            }
            else if(arr[mid]==1){
                mid++;
            }
            else{
                arr[mid]=arr[high];
                arr[high]=2;
                high--;
            }
        }
        return;
};
